src/firefly/js/templates/lightcurve/LCPanels.css

```diff
@@ -0,0 +1,26 @@
+.PanelResizableStyle {
+    width: 400px;
+    min-width: 400px;
+    height: 300px;
+    min-height: 300px;
+    overflow: auto;
+    background-color: #b3edff;
+    padding: 2px;
+    position: relative
+}
+
+.Header {
```

Comment:
CSS style name should follow BEM guildline: http://getbem.com/introduction/
Generic name like this may clash with others.
We use '__' for blocks separator and '--' for modifiers.
ie.  LCPanels__Header[--if-needed]